hathor/event/event_manager.py

```diff
@@ -85,9 +86,11 @@ def __init__(
     def start(self, peer_id: str) -> None:
         self._peer_id = peer_id
         self._event_ws_factory.start()
+        self._is_running = True
```

Comment:
Should we assert `self._is_running is False`?

Reply:
It wouldn't be a problem since the operation is idempotent, but I've added it to prevent human errors.

Reply:
Got it. Thanks!